File: src/my_robot_tutorials/scripts/robot_news_radio_transmitter.py
#!/usr/bin/env python

import rospy
from std_msgs.msg import String

if __name__ == '__main__':
    #if anonymous is True, node name is composed with suffix random numbers
    rospy.init_node('robot_news_radio_transmitter',anonymous=True) #unique node name
    pub = rospy.Publisher("/robot_news_radio",String,queue_size=10) #topic_name, message type and queue size

    rate = rospy.Rate(2)
    while not rospy.is_shutdown():
        msg = String()
        msg.data = "Hi, this is Msg from Robot news Radio !"
        pub.publish(msg)
        rate.sleep()
    
    rospy.loginfo("Node was stopped")

File: src/my_robot_tutorials/src/robot_news_radio_transmitter.cpp
#include <ros/ros.h>
#include <std_msgs/String.h>

int main(int argc,char *argv[])
{
    ros::init(argc,argv,"robot_news_radio_transmitter",
        ros::init_options::AnonymousName);
    ros::NodeHandle nh;
    ros::Publisher pub = nh.advertise<std_msgs::String>("/robot_news_radio",10);

    ros::Rate rate(3);

    while(ros::ok())
    {
        std_msgs::String msg;
        msg.data = "Hi, this is msg++ from Robot new radio";
        pub.publish(msg);
        rate.sleep();
    }
    
}